snippets/csharp/System.Net/WebClient/Overview/useragent.cs

```diff
@@ -5,25 +5,25 @@
 
 public class Test
 {
-    public static void Main (string[] args)
+    public static void Main(string[] args)
     {
         if (args == null || args.Length == 0)
         {
-            throw new ApplicationException ("Specify the URI of the resource to retrieve.");
+            throw new ApplicationException("Specify the URI of the resource to retrieve.");
         }
-        WebClient client = new WebClient ();
+        using WebClient client = new WebClient();
 
         // Add a user agent header in case the
         // requested URI contains a query.
 
-        client.Headers.Add ("user-agent", "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.2; .NET CLR 1.0.3705;)");
+        client.Headers.Add("user-agent", "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.2; .NET CLR 1.0.3705;)");
 
-        Stream data = client.OpenRead (args[0]);
-        StreamReader reader = new StreamReader (data);
-        string s = reader.ReadToEnd ();
-        Console.WriteLine (s);
-        data.Close ();
-        reader.Close ();
+        using Stream data = client.OpenRead(args[0]);
+        StreamReader reader = new StreamReader(data);
+        string s = reader.ReadToEnd();
+        Console.WriteLine(s);
+        data.Close();
```

Comment:
You added a `using` for data... why is the data.Close() still needed?  Also, why add a using for the Stream but not for the StreamReader?  The StreamReader will itself by default close the underlying stream, so this could instead just be:
```C#
using StreamReader reader = new StreamReader(client.OpenRead(args[0]));
```
and then you don't need the line above or either of the .Close calls.